admin/src/main/resources/io/buoyant/admin/js/spec/RouterClientsSpec.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
+
+define([
+  'jQuery',
+  'lodash',
+  'src/router_clients',
+  'src/metrics_collector',
+  'spec/fixtures/metrics',
+  'template/compiled_templates'
+], function($, _, RouterClients, MetricsCollector, metricsJson, templates) {
+  describe("RouterClients", function() {
+    var collector;
+    var $container = $("<div />");
+    var $clientsEl;
+    var $combinedClientGraphEl;
+
+    var initialRouterData = {
+      adder: {
+        clients: ["$/inet/127.1/9030", "$/inet/127.1/9029", "$/inet/127.1/9092"]
+      },
+      divider: {
+        clients: [
+          "$/inet/127.1/9031",
+          "$/inet/127.1/9032",
+          "$/inet/127.1/9033",
+          "$/inet/127.1/9034",
+          "$/inet/127.1/9035",
+          "$/inet/127.1/9036"
+        ]
+      },
+      multiplier: {
+        clients: [
+          "$/inet/127.1/9030",
+          "/%/io.l5d.port/4141/#/io.l5d.fs/foo",
+          "does-not-compute/",
+          "does_not_obey_rules"
+        ]
+      },
+      nothing_here: { clients: [] }
+    }
+
+    beforeEach(function () {
+      collector = MetricsCollector(metricsJson);
```

Comment:
Given that the collector api is pretty small, I'm wondering if we should just use a stub here instead of loading up the metricsJson fixtures. Something like
```
{
        start: function() {},
        registerListener: function() {},
        deregisterListener: function() {},
        onAddedClients: function() {},
}
```
 Wdyt?